dynamiqs/qarrays/qarray.py

```diff
@@ -210,6 +211,10 @@ def mT(self) -> QArray:
     def ndim(self) -> int:
         return len(self.shape)
 
+    @property
+    def _data(self) -> Array:
+        pass
```

Comment:
If possible can we avoid this? Or give it a weirder name, `data` is not explicit enough IMO.